packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -459,83 +457,67 @@ class HeapTreeViewState extends State<HeapTree>
   Widget _buildSnapshotControls(TextTheme textTheme) {
     return Row(
       children: [
-        Tooltip(
-          message: 'Take snapshot',
+        ActionButton(
```

Comment:
an action button with a child that is an outline button seems surprising. Can we just have an ActionButton?

Reply:
The ActionButton class gets us a tooltip, and sets it up w/ some consistent app defaults around tooltip delay. Besides the tooltip text it just takes a child, which can be anything.